src/AutoComplete.js

```diff
@@ -3,7 +3,7 @@ import createComponent from './createComponent'
 import mapError from './mapError'
 
 export default createComponent(AutoComplete, ({
-  input: { onChange, value },
+  input: { onChange, value, onUpdateInput },
```

Comment:
`onUpdateInput` is not in `input`. `input` are callbacks that `redux-form` generates.